lib/bowling.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'gif_client'
+require "gif_client"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

```suggestion
require 'gif_client'
```